<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GirisWidget</class>
 <widget class="QWidget" name="GirisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="giris_yap_verticalLayout_ana">
   <item>
    <spacer name="verticalSpacer_ust">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QFrame" name="giris_yap_form_frame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
  <property name="styleSheet">
   <string notr="true">#giris_yap_form_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
    padding: 20px;
}
QLabel {
    color: #ffffff;
    font-family: 'Segoe UI', sans-serif;
}
QLineEdit {
    background-color: #363a45;
    border: 1px solid #3c404d;
    border-radius: 5px;
    padding: 10px;
    color: white;
    font-size: 14px;
}
QLineEdit:focus {
    border: 1px solid #4facfe;
}</string>
  </property>
     <layout class="QVBoxLayout" name="verticalLayout_21">
      <item>
       <widget class="QLabel" name="giris_yap_baslik">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Giriş Yap&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="kullanici_adi_input_sayfa">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Kullanıcı Adı</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sifre_input_sayfa">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Şifre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="giris_yap_butonu_sayfa">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #007bff;
    color: white;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #0056b3;
}
QPushButton:pressed {
    background-color: #004085;
}</string>
        </property>
        <property name="text">
         <string>Giriş Yap</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QPushButton" name="sifre_unuttum_butonu_sayfa">
          <property name="styleSheet">
           <string notr="true">background-color: transparent; text-align: left; padding: 0;</string>
          </property>
          <property name="text">
           <string>Şifremi Unuttum?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="kayit_ol_butonu_sayfa">
          <property name="styleSheet">
           <string notr="true">background-color: transparent; text-align: right; padding: 0;</string>
          </property>
          <property name="text">
           <string>Hesabın Yok mu? Kayıt Ol</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
